packages/apache-annotator/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "apache-annotator",
+  "version": "0.1.0",
+  "description": "Web Annotation selector for engine.",
```

Comment:
Maybe pull the description from the root package.json. The root one might not even need a description.

Reply:
ah yes; I just copied the `package.json` from the selector package. We should probably give that one a less cryptic description some day. :)
I see that while copying I also left its dependencies, and pointer to its type declaration file; I removed both.